### DISCUSSION TITLE
Add static value to panel (or add data to source)

### DISCUSSION THREAD

**Discussion by Ropper1** [teslamate-org/teslamate#3456] [Q&A]
I’m on a leasing contract limiting my yearly mileage. I use the Drive stats to leep track on how my mileage develops over the 3 years for the contract. The panel <Extrapolated annual> calculates the panel <Driven> for the selected timespan. But my database was created a month after I recieved the car, so the first approx. 1.000 km is not in the database. 
is there a way to add those km into the database or maybe to add a static statement to the panel, basically saying “add 1.000 km to the calculation?

**Comment by DrMichael** [comment]
You can definitely change the query to add 1000km. Will be gone after next update, so store it outside.
#
